doc/manual/noteworthy-differences.rst

```diff
@@ -117,7 +117,8 @@ noteworthy differences:
 - In Julia, varargs are specified using the splat operator ``...``, which
   always follows the name of a specific variable, unlike R, for which ``...``
   can occur in isolation.
-- In Julia, modulus, is :obj:`%`, not ``%%``.
+- In Julia, modulus, is ``mod(a, b)``, not ``a %% b``. ``%`` in Julia is the
```

Comment:
could you open a new version of this against master, and delete the unnecessary comment after modulus? Thanks!